diff --git target/catkin_ws/src/ros_comm/xmlrpcpp/src/XmlRpcDispatch.cpp target/catkin_ws/src/ros_comm/xmlrpcpp/src/XmlRpcDispatch.cpp
index 95ed346..2698624 100644
--- target/catkin_ws/src/ros_comm/xmlrpcpp/src/XmlRpcDispatch.cpp
+++ target/catkin_ws/src/ros_comm/xmlrpcpp/src/XmlRpcDispatch.cpp
@@ -6,6 +6,7 @@
 #include <math.h>
 #include <errno.h>
 #include <sys/timeb.h>
+#include <sys/select.h>
 
 #if defined(_WINDOWS)
 # include <winsock2.h>
